data/human/derelicts.txt

```diff
@@ -0,0 +1,1793 @@
+				goto success
+			label powersurge
+			`	With a few simple repairs you manage to get the ship's power systems operational. Briefly. The power shuts down again with a violent vibration shuddering through the ship. You quickly run back to your ship before the <npc> breaks apart.`
+				depart
+			label crumple
+			`	As you are patching up a segment of hull you notice that a deformation in it is slowly expanding, worse this is a structurally essential part of the hull. You quickly run back to your ship before the <npc> breaks apart.`
+				depart
+			label misfire
+			`	As you are fixing the engine systems a valve bursts open, and with no power the secondary valve cannot close itself off, letting coolant uncontrollably fly into the hull of the ship. The force of the liquid sends a violent vibration shuddering through the ship. You quickly run back to your ship before the <npc> breaks apart.`
+				depart
+			label flee
+			action
+				payment 7500
+			`	Wary of your own foolish repairs you decide to leave the ship alone, however you did manage to hang onto a little bit of scrap worth <payment> you found during your repairs.`
+				depart
+			label fail
+			`	As the ship begins to thrum to life, a loud explosion rocks the ship!`
+			choice
+				`	(Run!)`
+			`	You start running through the tight corridors of the ship, more explosions threatening to sweep you off your feet, and objects tumbling everywhere.`
+			choice
+				`	(Run into the airlock.)`
+					goto run
+				`	(Dive for the airlock.)`
+					goto dive
+				`	(Slide into the airlock.)`
+			branch object
+				random < 5
+			`	As you slide across the floor, explosions erupt on both sides of you sending chunks of metal flying above your head, and you're sure that you look like an action hero as you careen through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label run
+			branch object
+				random < 10
+			`	As you run towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you barge through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label dive
+			branch object
+				random < 10
+			`	As you dive towards the airlock, explosions erupt on both sides of you, and you're sure that you look like an action hero as you tuck and roll through the airlock to safety. Having narrowly escaped death and not wishing to chance it, you quickly depart.`
+				depart
+			label object
+			`	Before you can make it to the airlock something comes flying straight towards you.`
+			choice
+				`	(Try to block.)`
+					goto block
+				`	(Karate chop it.)`
+					goto ninja
+				`	(Close your eyes.)`
+					goto dead
+			label dead
+			`	The object strikes you, sending you sprawling as the ship explodes around you, resulting in your doom.`
+				die
```

Comment:
What is, instead of you being killed by the ship, there derelict turns out to be a pirate trap? In EVN, a derelict could be a trap, have survivors you could transport for payment, or just be empty and loot-able.